src/cfnlint/helpers.py

```diff
@@ -290,15 +293,36 @@ def onerror(os_error):
         raise os_error
 
     for root, _, filenames in os.walk(directory, onerror=onerror):
-        for filename in fnmatch.filter(filenames, '[A-Za-z]*.py'):
-            pluginname = filename.replace('.py', '')
-            try:
-                fh, filename, desc = imp.find_module(pluginname, [root])
-                mod = imp.load_module(pluginname, fh, filename, desc)
-                result.extend(create_rules(mod))
-            finally:
-                if fh:
-                    fh.close()
+        if sys.version_info < (3,):
+            for filename in fnmatch.filter(filenames, '[A-Za-z]*.py'):
+                pluginname = filename.replace('.py', '')
+                try:
+                    fh, filename, desc = imp.find_module(pluginname, [root])
+                    mod = imp.load_module(pluginname, fh, filename, desc)
+                    result.extend(create_rules(mod))
+                finally:
+                    if fh:
+                        fh.close()
+        else:
+            loader_details = (
+                importlib.machinery.SourceFileLoader,  # pylint: disable=no-member
```

Comment:
I'd recommend against pylint since it makes you do write all these hoop jumping comments

100% agree with Luke Plant's blog post here https://lukeplant.me.uk/blog/posts/pylint-false-positives/

Reply:
I'll take a look into what this change could look like.  Thanks for the recommendation.